### DISCUSSION TITLE
Is it okay to add translations of readme.md to Korean and Japanese?

### DISCUSSION THREAD

**Discussion by foreverfl** [remarkjs/remark#1364] [Q&A]
This is my first contribution to an open source project.

I would like to translate the description on this page, add buttons for the translated pages, and send a PR for merging.

Would that be possible?

**Comment by ChristianMurphy** [comment]
Welcome to the unified and remark community @foreverfl! We're happy to have you.

> This is my first contribution to an open source project.

Welcome to contributing to open source!
How did you hear about remark? Is it something you are using on a project?

> I would like to translate the description **on this page**

Which page is the page you are referring to?

> I would like to **translate the description** on this page [...] to Korean and Japanese

Translations are something I'm open to, but cautious about, and personally wouldn't recommend as a first contribution to remark/unified.

My caution is I am not fluent in either of these languages and wouldn't really be able to review.
And I'm cautious approving any changes which I don't understand and couldn't maintain myself.
Perhaps others from @remarkjs/contributors would be more comfortable?

---

Another option for a first contribution would be looking through issues labelled help wanted https://github.com/search?q=org%3Aremarkjs+label%3A%22help+wanted+%F0%9F%99%8F%22+is%3Aopen&type=issues&ref=advsearch
And seeing if there are any you would be interested and comfortable solving.

**Reply by wooorm** [reply]
I am also very cautious. The hard part of open source is *maintaining* things. Not writing them once.
To accept translations, I think we’d need an active community around that language first.

unified is also 100s of projects. An immense job. Already hard to keep it all up to date in English.

**Reply by remcohaszing** [reply]
I prefer not to add translations. It’s hard to keep translations up-to-date for languages the maintainers speak. It’s even harder for languages we don’t speak. Automated translations by a browser extension are probably better than manual unmaintained translations.

**Reply by foreverfl** [reply]
I have used `react-markdown` in the blog I developed, and when customizing it, I used `remark`.

After reading all the comments, I understood how difficult it is to keep the documentation up to date.

For now, I have forked some of the remark/rehype repositories.

The translation of the documentation will be done in my forked repositories.

If there is something I want to contribute, I'll send a PR.

Thanks for answering my question.
